msbuild/Xamarin.iOS.Tasks.Core/Xamarin.TVOS.AppExtension.CSharp.targets

```diff
@@ -22,7 +22,9 @@ Copyright (C) 2014-2016 Xamarin. All rights reserved.
 		<TargetFrameworkIdentifier Condition="'$(TargetFrameworkIdentifier)' == ''">Xamarin.TVOS</TargetFrameworkIdentifier>
 		<TargetFrameworkVersion Condition="'$(TargetFrameworkVersion)' == ''">v1.0</TargetFrameworkVersion>
 
-		<DefineConstants>__UNIFIED__;__MOBILE__;__TVOS__;$(DefineConstants)</DefineConstants>
+		<DefineConstants Condition="!$(DefineConstants.Contains('__UNIFIED__'))">__UNIFIED__;$(DefineConstants)</DefineConstants>
+		<DefineConstants Condition="!$(DefineConstants.Contains('__MOBILE__'))">__MOBILE__;$(DefineConstants)</DefineConstants>
+		<DefineConstants Condition="!$(DefineConstants.Contains('__TVOS__'))">__TVOS__;$(DefineConstants)</DefineConstants>
```

Comment:
Agreed, but that's kind of a corner case and in that case you can easily define __UNIFIED__ too. I'm open to suggestions on something that will not make this impossible to read and still not duplicate :)